modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -0,0 +1,777 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-about-iam-resources.adoc
+
+[id="rosa-sts-account-wide-roles-and-policies_{context}"]
+= Account-wide IAM role and policy reference
+
+This section provides details about the account-wide IAM roles and policies that are required for ROSA deployments that use STS, including the Operator policies. The JSON files that define the policies are also provided.
+
+The account-wide roles and policies are specific to an OpenShift version, for example OpenShift 4.8. You can minimize the required STS resources by reusing the account-wide roles and policies for multiple clusters of the same version.
+
+[NOTE]
+====
+If your use case requires it, you can deploy multiple sets of account-wide IAM roles and policies for a cluster version by specifying different prefixes for each set.
+====
+
+.ROSA installer role, policy, and policy files
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-Installer-Role`
+|An IAM role used by the ROSA installer.
+
+|`ManagedOpenShift-Installer-Role-Policy`
+|An inline IAM policy that provides the ROSA installer with the permissions required to complete cluster installation tasks.
+
+|===
+
+[NOTE]
+====
+The IAM role and policy names include the role prefix that is specified when the STS resources are created. The resource names in the examples in this section include the default prefix `ManagedOpenShift`.
+====
+
+.`sts_installer_trust_policy.json`
+[%collapsible]
+====
+[source,yaml]
+----
+"Effect": "Allow",
+"Principal": {
+    "AWS": [
+        "arn:aws:iam::123456789012:role/RH-Managed-OpenShift-Installer"
```

Comment:
I wonder if we should be referencing the actual account ID for the managed installer jump role (`710019948333`). This account ID will be the same for all customers in the production environment.